src/main/db_config.cpp

```diff
@@ -1,3 +1,4 @@
+#pragma clang diagnostic push
```

Comment:
Why do we need this line? Don't you need `pop` somewhere?

Reply:
Generated by ide, removed